### DISCUSSION TITLE
Add to F-droid

### DISCUSSION THREAD

**Discussion by jluttine** [Emurgo/yoroi-mobile#1918] [Ideas]
Could you make the app compatible with F-droid and add it there too? That'd be superb!

**Comment by v-almonacid** [comment]
Hi @jluttine, thanks for your suggestions. While some of us like very much F-droid, unfortunately we are not considering releasing there at the moment - probably we'll do it in the near future. 
Any particular reason why you wanted the app there?

**Comment by jluttine** [comment]
Thanks for the reply! My personal reason is that I prefer and only use F-Droid because I don't have (and I don't want to have) Google Play Store nor any other Google Spyware on my phone. If the app can be added to F-Droid and I install from there, I know that the app satisfies the FOSS requirements of F-Droid and some other nice properties that follow like independence of Google Spyware.

**Comment by ghost** [comment]
Are there any plans to revisit this?

**Comment by v-almonacid** [comment]
@wylel unfortunately we have other priorities at the moment (ie. support for the Mary hard fork). But I'll add this to our TODO list to make sure we take a look as soon as we have time for it.
